pillar/flavors/pico.sls

```diff
@@ -45,6 +45,9 @@
             "parallelism": 1,
             "taskmanager_mem_preallocate": false,
             "pyflink_yarn_container_count": 1
+        },
+        "graphite-api":{
+            "retentions": "60:10080"
```

Comment:
Can we use the singular form here 'retention' and note what it is a retention for in the name (i.e. apps)

Pico should have the lowest retention of the three flavors. A day is sufficient.

Reply:
Changed 'retentions' to 'retention_spark_metrics'.
For pico, have set retention a day.